Moonscraper Chart Editor/Assets/Scripts/Game/Controllers/TimelineMovementController.cs

```diff
@@ -17,6 +17,7 @@ public class TimelineMovementController : MovementController
     Texture2D dragCursorNegative;
 
     Vector2? middleClickDownScreenPos = null;
+    CursorMode middleClickCursorMode = CursorMode.ForceSoftware;
```

Comment:
This could be compile-time determined using `ifdef UNITY_STANDALONE_LINUX` instead of using the runtime platform.